boards/posix/native_posix/doc/board.rst

```diff
@@ -519,6 +519,20 @@ The following peripherals are currently provided with this board:
   must be powered down and support Bluetooth Low Energy (i.e. support the
   Bluetooth specification version 4.0 or greater).
 
+**Display driver**:
+  A display driver is provided that will create a window on the host machine to
```

Comment:
How about:
```
**Display driver**:
  A character frame buffer (CFB) display driver is provided that
  creates a window on the host machine to render display content. 
  (Currently, the CFB framework is limited to monochrome
  vertical tiled displays such as found on the :ref:`reel_board``.)

  This driver requires a 32-bit version of  the`SDL2`_ library
  on the host machine and ``pkg-config`` settings to correctly
  pickup the SDL2 install path and compiler flags.

  On a Ubuntu 18.04 host system, for example,  install the
  ``pkg-config`` and ``libsdl2-dev:i386`` packages, and configure
  the pkg-config search path with these commands::

     $ sudo apt-get install pkg-config libsdl2-dev:i386
     $ export PKG_CONFIG_PATH=/usr/lib/i386-linux-gnu/pkgconfig

```

Also, the picture you have in the commit message would be a nice to show here.  (I see though that you used a proportional font in that display.  Does the CFB driver handle proportional fonts or are only mono-spaced fonts supported?)

Reply:
Thx for the proposal looks much better, will update the docs.

> Also, the picture you have in the commit message would be a nice to show here. (I see though that you used a proportional font in that display. Does the CFB driver handle proportional fonts or are only mono-spaced fonts supported?)

This picture that is shown is actually for the lvgl sample application which is part of PR #6826.
Once CFB came available with the introduction of the reel board it looked like a better idea to reuse that sample.

But I could make a screenshot of CFB in action but do you have any idea on the policy of putting binary files in to the repo?